Makefile

```diff
@@ -62,7 +62,7 @@ BUNDLE_DIR := bundle
 BUNDLE_MANIFEST_DIR := $(BUNDLE_DIR)/manifests
 BUNDLE_IMG ?= quay.io/external-dns-operator/external-dns-operator-bundle:latest
 CATALOG_DIR := catalog
-OCP_VERSION ?= 4.21
+OCP_VERSION ?= 4.19
```

Comment:
This change is not needed anymore. The test is supposed to be run on 4.19 as ITS is not linked to 4.19 FBC.